README.md

```diff
@@ -77,49 +84,52 @@ Given the example above, the code would look like this:
+If you look at the URL in the address bar, you'll see your Client ID. On the previous page, the **Sign In with GitHub** button was in fact a link to an OAuth Authorization page, with your Client ID as a parameter. Sinatra built this link for us when we passed the `CLIENT_ID` into the index.erb template on line 24 of `server.rb`. Sinatra then interpolated our Client ID into the OAuth API link on line 15 of `index.erb`.
 
 Go ahead and Authorize the Application access to your GitHub account data, and you should be redirected to your callback page: `/profile`.
 
 ![success](https://user-images.githubusercontent.com/3988879/40751094-f98ceea2-6426-11e8-915a-c44e0f38cfb6.png)
 
-After a successful Application authorization, GitHub provides us with a temporary ***Authorization Grant Code*** (see it in the URL up there?). We need to `POST` this code to GitHub's `https://github.com/login/oauth/access_token` endpoint to recieve an `access_token`. Once we recieve an Access Token, we can interact with account data.
+After a successful Application authorization, GitHub provides us with a temporary *authorization grant code* (see it in the URL up there?). We need to `POST` this code to the's `/login/oauth/access_token` endpoint to recieve an *access_token* The access token is what grants us access to the account data that we want to display.
 
-To simplify the process of `POST`ing the Authorization Grand Code back to GitHub, we'll use the ruby gem [rest-client](https://github.com/rest-client/rest-client).
+To simplify the process of `POST`ing the authorization grant code back to GitHub, we'll use the [rest-client](https://github.com/rest-client/rest-client) Ruby gem.
 
-First, let's update our `/profile` view to retrieve the temporary Authorization Grant Code. We'll use Sinatra's built in webserver interface, [Rack](https://rack.github.io/) to grab the code from the session data.
+First, let's update our `/profile` view to retrieve the temporary authorization grant code. We'll use Sinatra's built in webserver interface, [Rack](https://rack.github.io/) to grab the code from the session data.
 
 ``` ruby
 get '/profile' do
-    # Retrieve temporary Authorization Grant Code
+    # Retrieve temporary authorization grant code
     session_code = request.env['rack.request.query_hash']['code']
     
     erb :profile
 end
 ```
 
-Now that we have the Authorization Grant Code, let's use the `rest-client` gem to `POST` it back to GitHub along with our `CLIENT_ID` and `CLIENT_SECRET` in exchange for our `access_token`.
+Now that we have the authorization grant code, let's use `rest-client` to `POST` it back to GitHub along with our `CLIENT_ID` and `CLIENT_SECRET` in exchange for our `access_token`.
+
+[DEGW: Why doesn't octokit support this? That seems a little sad. Although I suppose the reason is because it's redundant if you're using something like `omniauth`.]
```

Comment:
Octokit doesn't support OAuth, as it's actually a separate API. Abstracting the OAuth process further with a gem is a good idea for this guide however 👍

Reply:
Haven't looked too closely, so apologize if this is of-base, but is this not what this [oauth access tokens section](https://github.com/octokit/octokit.rb#oauth-access-tokens ) is doing?

Reply:
That appears to be exactly what it does! I hadn't actually read the that part of Readme. Thanks for spotting this!